src/emcpy/plots/create_plots.py

```diff
@@ -407,7 +408,7 @@ def _map_scatter(self, plotobj, ax):
 
             cs = ax.scatter(plotobj.longitude, plotobj.latitude,
                             c=plotobj.data, s=plotobj.markersize,
-                            **inputs, norm=norm, transform=self.projection.projection)
+                            **inputs, norm=norm, transform=ccrs.PlateCarree())
```

Comment:
I believe this was from a different change that got mixed with this PR. This needs to be changed though and I can show reasons in a future PR.

Reply:
This looks like the thing that Jakir was working on for polar projections. It needs to support multiple projections.

Reply:
I'll revert the changes in this PR. However, I will show issues it creates I found when adding new features in a separate PR.